src/components/auth.tsx

```diff
@@ -22,12 +23,13 @@ import { UserAvatar } from '@components/user-avatar';
 const Dropdown: React.FC<BoxProps & { onSignOut?: () => void; show?: boolean }> = memo(
   ({ onSignOut, show }) => {
     return (
-      <Fade in={show}>
+      <Fade in={true}>
```

Comment:
🤔 ?

Reply:
O, that was just for testing! Sorry, will put back.